README.md

```diff
@@ -1110,6 +1110,12 @@ IEEE International Conference on Multisensor Fusion and Integration for Intellig
 *[X-maps: Direct Depth Lookup for Event-based Structured Light Systems](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Morgenstern_X-Maps_Direct_Depth_Lookup_for_Event-Based_Structured_Light_Systems_CVPRW_2023_paper.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2023. [Project page](https://fraunhoferhhi.github.io/X-maps/), [Code](https://github.com/fraunhoferhhi/X-maps).  
 
+<a name="voxel-mono"></a>
+### Monocular Voxel Reconstruction
```

Comment:
Actually I have been contemplating this topic for a while now. Currently, most event-based 3D reconstruction methods use depth estimation to obtain a 3D representation. However, the depth estimation step may not be necessary, as it is possible to extract features from the raw events and use them to create a 3D representation, as demonstrated in our paper. Other works also exist that convert event data to silhouettes and use them to create a 3D representation.

Therefore, I believe a new taxonomy for event-based 3D reconstructions may be necessary. Personally, I think that starting with a taxonomy based on traditional image-based 3D reconstructions would be a good starting point, although I am still reviewing papers. Any thoughts or feedback on this would be greatly appreciated.

As it is late where I am (Sydney time 2 am), I will think of a more generic categorization title for this entry tomorrow.

Reply:
How about:
```markdown
<a name="e2e-mono"></a>
### Monocular End-to-end 3D Reconstruction
```

Reply:
I think it is better to write the target task (object reconstruction) rather than how it is achieved (end-to-end).

Reply:
Sure, that sounds good too, I'll change accordingly now. Thanks for the suggestion!